engines/order_management/app/services/order_management/order/updater.rb

```diff
@@ -20,6 +20,10 @@ def initialize(order)
       # associations try to save and then in turn try to call +update!+ again.)
       def update
         update_all_adjustments
+        update_totals_and_states
+      end
+
+      def update_totals_and_states
```

Comment:
:100:

Reply:
This little refactor opens up a world of possibilities for updating the order's totals and states without recalculating all adjustments, and there are a lot of places where we can do that. Eg: places where we currently call `order.update!` but there's no actually need for all the adjustments to be updated. :rocket: